src/components/Comments.jsx

```diff
@@ -1,17 +1,46 @@
 import { useEffect, useState } from "react";
-import { getComments } from "../../api";
+import { getComments, postComment } from "../../api";
 
 export const Comments = ({article_id}) => {
     const [comments, setComments] = useState([])
+    const [commentInput, setCommentInput] = useState("")
     
     useEffect(() => {
+        // if(newComment != ""){
+        //     postComment(article_id, newComment).then((data) => setComments(data, ...comments))
+        //     setNewComment("");        
+        // }
         getComments(article_id).then((data) => {
             setComments(data)
         })
-    },[article_id])
+    },[article_id, comments])
```

Comment:
Article id isn't going to change unless the whole page changes as you're getting that information from the url. By adding comments here you're causing an entire reload every time a comment is added. Is that what you want?